pubtools/_pulp/tasks/set_maintenance/set_maintenance_off.py

```diff
@@ -0,0 +1,48 @@
+import logging
+import re
+
+from .base import SetMaintenance
+
+
+LOG = logging.getLogger("set-maintenance")
+
+
+class SetMaintenanceOff(SetMaintenance):
+    """Unset repositories maintenance mode.
+
+    See "pub maintenance-on --help" for more information on maintenance mode.
+    """
+
+    def run(self):
+        report = self.pulp_client.get_maintenance_report().result()
+
+        to_remove = []
+        if self.args.repo_regex:
+            for entry in report.entries:
+                if re.match(self.args.repo_regex, entry.repo_id):
+                    to_remove.append(entry.repo_id)
+        if self.args.repo_ids:
+            existed_repo_ids = [entry.repo_id for entry in report.entries]
+            for repo_id in self.args.repo_ids:
+
+                if repo_id in existed_repo_ids:
+                    to_remove.append(repo_id)
+                else:
+                    LOG.warn("Repository %s is not in Maintenance Mode", repo_id)
```

Comment:
I don't think we need Maintenance Mode capitalized like this.

```suggestion
                    LOG.warn("Repository %s is not in maintenance mode", repo_id)
```